cpu/stellaris_common/license.html

```diff
@@ -0,0 +1,364 @@
+<html>
```

Comment:
not sure, but this file does not read like BSD style licensing. Though there is also the EULA stuff included, which does. So which is the correct license for the stellaris code? As I see it -> The stellaris headers include alreday a BSD license header, so there is no reason to keep this file as well as the EULA*.txt file. @OlegHahm, do you have an opinion on this?